pkg/kubernetes/watcher.go

```diff
@@ -138,7 +140,8 @@ func NewObjListWatcher(bpfSupportedMetrics bpf.SupportedMetrics) *ObjListWatcher
 		},
 	})
 	if err != nil {
-		klog.Fatalf("%v", err)
+		klog.Errorf("%v", err)
+		return nil
```

Comment:
```suggestion
		return err
```

Reply:
I didn't change the function to return errors:

```
func NewObjListWatcher(bpfSupportedMetrics bpf.SupportedMetrics) *ObjListWatcher
```

which is why I was returning nil, I can update the function to also return the error, WDYT?